frontend/packages/console-app/src/components/quick-starts/loader/QuickStartsLoader.tsx

```diff
@@ -1,28 +1,29 @@
 import * as React from 'react';
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import { referenceForModel } from '@console/internal/module/k8s/k8s';
 import { QuickStart } from '../utils/quick-start-types';
-import { getQuickStarts } from '../utils/quick-start-utils';
+import { QuickStartModel } from '../../../models';
 import QuickStartPermissionChecker from './QuickStartPermissionChecker';
 
 type QuickStartsLoaderProps = {
   children: (quickStarts: QuickStart[], loaded: boolean) => React.ReactNode;
 };
 
 const QuickStartsLoader: React.FC<QuickStartsLoaderProps> = ({ children }) => {
-  const [quickStarts, setQuickStarts] = React.useState<QuickStart[]>([]);
+  const [quickStarts] = useK8sWatchResource<QuickStart[]>({
+    kind: referenceForModel(QuickStartModel),
+    isList: true,
+  });
   const [allowedQuickStarts, setAllowedQuickStarts] = React.useState<QuickStart[]>([]);
   const [loaded, setLoaded] = React.useState<boolean>(!(quickStarts.length > 0));
```

Comment:
`useK8sWatchResource` hook returns a loaded variable to track loading of resource. use that.

Reply:
using quickStartsLoaded